api/http/handler/hostmanagement/fdo/configure.go

```diff
@@ -33,26 +35,12 @@ func (payload *deviceConfigurePayload) Validate(r *http.Request) error {
 		return errors.New("the device name cannot be empty")
 	}
 
-	if err := validateURL(payload.ProfileURL); err != nil {
```

Comment:
Don't you need to add a validation for `ProfileID` here?

Reply:
done